infra/gcp/lib.sh

```diff
@@ -228,6 +231,32 @@ function empower_group_as_viewer() {
         --role roles/viewer
 }
 
+# Grant privileges to prow in a staging project
+# $1: The GCP project
+# $2: The GCS scratch bucket
+function empower_prow() {
+    if [ $# -lt 2 -o -z "$1" -o -z "$2" ]; then
+        echo "empower_prow(project, bucket) requires 2 arguments" >&2
+        return 1
+    fi
+    project="$1"
+    bucket="$2"
+
+    # Allow prow to trigger builds.
+    gcloud \
+        projects add-iam-policy-binding "${project}" \
+        --member "serviceAccount:${PROW_SVCACCT}" \
+        --role roles/cloudbuild.builds.builder
+
+    # Allow prow to access build logs.
```

Comment:
tiny nit: `objectViewer` allows prow to access build logs; `objectCreator` allows it to provide the build source to GCB